controllers/operands/kubevirtConsolePlugin_test.go

```diff
@@ -279,6 +284,50 @@ var _ = Describe("Kubevirt Console Plugin", func() {
 					foundResource),
 			).ToNot(HaveOccurred())
 
+			Expect(foundResource.Spec.Replicas).ToNot(Equal(outdatedResource.Spec.Replicas))
+			Expect(foundResource.Spec.Replicas).To(Equal(expectedResource.Spec.Replicas))
+			Expect(foundResource.Spec.Template.Spec.Containers[0].Image).To(Equal(expectedResource.Spec.Template.Spec.Containers[0].Image))
+			Expect(reflect.DeepEqual(expectedResource.Spec, foundResource.Spec)).To(BeTrue())
```

Comment:
I think we don't need the first check because of the second one, no?
```suggestion
Expect(reflect.DeepEqual(expectedResource.Spec, foundResource.Spec)).To(BeTrue())
```

Reply:
yes, it's already fully covered by the second line, it's just a matter of visibility